src/tlf.h

```diff
@@ -189,12 +190,14 @@ typedef struct {
 struct qso_t {
     char *logline;
     int band;
+    int bandindex;
```

Comment:
(can't comment on unchanged line)
```char exchange[13];```  should be 24 ((77-55+1)+1) to be able to store a QSO-mode exchange

Reply:
You mean in declaration of worked_t? You are right - corrected.